internal/tiltfile/config/config_def.go

```diff
@@ -104,10 +104,16 @@ func (cd ConfigDef) parseArgs(args []string) (ret configMap, output string, err
 			continue
 		}
 		fs.Var(ret[name], name, def.usage)
+		// for bools, make "--foo" equal to "--foo true"
+		if _, ok := ret[name].(*boolSetting); ok {
+			fs.Lookup(name).NoOptDefVal = "true"
+		}
 	}
 
 	err = fs.Parse(args)
 	if err != nil {
+		_, _ = fmt.Fprintf(w, "Error parsing tiltfile config args: %v\nUsage:\n", err)
+		fs.PrintDefaults()
```

Comment:
pflag advertises itself as a drop-in replacement for flag, but it doesn't print usage on error by default (though it does if you set it to os.Exit or panic on error)